### PR TITLE
[KOGITO-3793] Review OLM UI spec descriptors

### PR DESCRIPTION
**Pull request by sutaakar**
Signed-off-by: Karel Suta <ksuta@redhat.com>

https://issues.redhat.com/browse/KOGITO-3793

porting https://github.com/kiegroup/kogito-operator/pull/988

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]
Same question here regarding how the UI looks like after this change.

File: api/v1/kogitobuild_types.go [hunk 1]
```diff
@@ -68,70 +71,83 @@ type KogitoBuildSpec struct {
...
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:label"
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Maven Mirror URL"
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:text"
 	MavenMirrorURL string `json:"mavenMirrorURL,omitempty"`
 
 	// Image used to build the Kogito Service from source (Local and Remote).
-	// The operator will use the one provided by the Kogito Team based on the "Runtime" field.
+	//
+	// If not defined the operator will use image provided by the Kogito Team based on the "Runtime" field.
```
Comment (radtriste) [inline api/v1/kogitobuild_types.go:97]: RHPAM Kogito Team ?

File: api/v1/kogitobuild_types.go [hunk 2]
```diff
@@ -68,70 +71,83 @@ type KogitoBuildSpec struct {
...
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:text"
 	// +optional
 	BuildImage string `json:"buildImage,omitempty"`
 
 	// Image used as the base image for the final Kogito service. This image only has the required packages to run the application.
+	//
 	// For example: quarkus based services will have only JVM installed, native services only the packages required by the OS.
-	// The operator will use the one provided by the Kogito Team based on the "Runtime" field.
+	//
+	// If not defined the operator will use image provided by the Kogito Team based on the "Runtime" field.
```
Comment (radtriste) [inline api/v1/kogitobuild_types.go:112]: RHPAM Kogito Team ?

**Review by radtriste — APPROVED** [review]
2 minor comments

File: api/v1/kogitobuild_types.go (hunk 1 above)
Comment (spolti) [reply api/v1/kogitobuild_types.go:97]: maybe just, "will use the default builder image"

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
just a minor comment.
Thanks @sutaakar

**Comment by sutaakar** [comment]
changes incorporated in https://github.com/kiegroup/rhpam-kogito-operator/pull/73, closing this PR

**Pull request closed without merging**